test/test_message_directory.py

```diff
@@ -0,0 +1,138 @@
+"""
```

Comment:
Don't put creation/authorship info into files that are controlled by a VCS. Leave that to the VCS.

Reply:
Should I copy the doc string from `ssm/message_directory.py` here (minus the creation/authorship info)?

Reply:
I'd actually use the type of header used here (i.e. inline comment syntax): https://github.com/apel/apel/blob/dev/test/test_norm_sum_record.py

But I would drop the "The" from our name (and use 2018 of course).

Putting it in the file's docstring means it can pop up on the command line and via other tools in ways we probably don't intend it to.

Reply:
changed